src/lib/storage/vector_compression/bitpacking/bitpacking_vector_type.hpp

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include "compact_vector.hpp"
+#include "types.hpp"
+
+namespace opossum {
+
+// Right now, compact_vector is only used in the BitpackingVector where the value type uint32_t is sufficient.
```

Comment:
Can you elaborate a little bit here? We only need `uint32_t` but there still is `uint64_t` being mentioned twice. What do these parameters do?

Reply:
documented all template arguments